src/app/templates/SendForm/index.tsx

```diff
@@ -25,8 +32,12 @@ const SendForm: FC<SendFormProps> = ({ assetSlug = 'tez' }) => {
 
   const { data: tokens = [] } = useDisplayedFungibleTokens(chainId, account.publicKeyHash);
   const { data: collectibles = [] } = useCollectibleTokens(chainId, account.publicKeyHash, true);
+  const assetsSortPredicate = useAssetsSortPredicate();
 
-  const assets = useMemo<IAsset[]>(() => ['tez' as const, ...tokens, ...collectibles], [tokens, collectibles]);
+  const assets = useMemo<IAsset[]>(
+    () => ['tez' as const, ...tokens, ...collectibles].sort((a, b) => assetsSortPredicate(getSlug(a), getSlug(b))),
```

Comment:
could we avoid assertion here?

and also, does it mean that we sort 'tez' by USD value? I guess tez need to be on top everytime

Reply:
Sorry, I can't see any assertion here. Did you confuse "assets" and "assert"? Also, TEZ token is always at the top (see https://github.com/madfish-solutions/templewallet-extension/pull/874/files#diff-01a90946e02c3c9ab437b5b0015b3512c721d2921c3ed599a88b6152e19b217fR304)

Reply:
In the sort function, we use conditions like `return 1 and return -1` for tezos, it is mean that tezos will always on the top

Reply:
I talk about this one: 
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions